app/src/main/java/com/sopt/gongbaek/presentation/ui/mypage/screen/MyPageContract.kt

```diff
@@ -16,18 +16,29 @@ class MyPageContract {
         val registerClosedGroups: List<GroupInfo> = listOf(),
         val applyGroupsLoadState: UiLoadState = UiLoadState.Idle,
         val applyActiveGroups: List<GroupInfo> = listOf(),
-        val applyClosedGroups: List<GroupInfo> = listOf()
+        val applyClosedGroups: List<GroupInfo> = listOf(),
+        val versionName: String = "",
```

Comment:
[3] viewModel init 블록으로 `getVersionInfo`호출하는 방법 대신 state에 초기값으로 부여하는건 어떤가요?!
```suggestion
        val versionName: String = BuildConfig.VERSION_NAME,
```